src/databricks_ai_bridge/vector_search_retriever_tool.py

```diff
@@ -2,6 +2,7 @@
 from typing import Any, Dict, List, Optional
 
 import mlflow
+from databricks.vector_search.utils import CredentialStrategy
```

Comment:
Also I am curious if we should include this globally just for the types. Maybe we can localize the import and no need to include the dep in project.toml for databricks-ai-bridge